nodes/text/debug_print.py

```diff
@@ -55,9 +53,19 @@ def draw_buttons_ext(self, context, layout):
         for i, socket in enumerate(self.inputs):
             layout.prop(self, "print_socket", index=i, text=socket.name)
 
+    def draw_socket_boolean(self, socket, context, layout):
+        text = f"{socket.name}. {SvGetSocketInfo(socket)}"
+        layout.label(text=text)
+        icon = ("HIDE_ON", "HIDE_OFF", )[self.print_socket[socket.index]]
+        layout.prop(self, "print_socket", icon=icon, index=socket.index, text="")
+
     def sv_update(self):
         multi_socket(self, min=1)
 
+        for socket in self.inputs:
+            if not socket.custom_draw:
+                socket.custom_draw = "draw_socket_boolean"
```

Comment:
This looks a bit suspicions to apply every update,

Reply:
because `sv_init` is only called once, and sockets are added dynamically by `multi_socket`. and they won't get the custom_draw string.  

this does nothing if the socket has a custom draw.

Reply:
putting that code inside `process()` is possible, but UI does not update immediately.. the user must interact with the blender UI in a way to triggers the process function. 

maybe i'm not understanding what's happening here, but it does seem that the node's process function is called if I hover over the label area of a socket (no clicking or anything..)

![socket_showing_activity_in_process](https://user-images.githubusercontent.com/619340/126832634-0cc98f65-40fa-401d-ab17-081e4ad8c0d3.gif)